codegen/runner/post-steps.sh

```diff
@@ -1,18 +1,36 @@
 #!/usr/bin/env bash
 set -e
+set -o pipefail
 
-if [ -z $1 ]; then
+if [ -z "$1" ]; then
 	echo "prefix argument (\$1) is missing"
 	exit 1
 fi
 
-PREFIX="$1"
-DIRNAME="$(dirname $0)"
-EASY_JSON_RAW_DIR="$DIRNAME/../../scripts/easy_json"
-EASY_JSON_DIR="`cd "${EASY_JSON_RAW_DIR}";pwd`"
-EASY_JSON_FILE="$EASY_JSON_DIR/easy_json.go"
-EASY_JSON_BINARY="$EASY_JSON_DIR/easy_json"
+if [ -z "$2" ]; then
+	echo "config dir argument (\$2) is missing"
+	exit 1
+fi
 
 start=$(cat .TMP_ZANZIBAR_TIMESTAMP_FILE.txt)
 
-# TODO : Add new post steps :)
+BUILD_DIR="$1"
+CONFIG_DIR="$2"
+
+DIRNAME="$(dirname "$0")"
+if [ -d "$DIRNAME/../../vendor" ]; then
+	MOCKERY_RAW_DIR="$DIRNAME/../../vendor/github.com/vektra/mockery"
+else
+	MOCKERY_RAW_DIR="$DIRNAME/../../../../../github.com/vektra/mockery"
+fi
+MOCKERY_DIR="$(cd "$MOCKERY_RAW_DIR";pwd)"
+MOCKERY_MAIN_FILE="$MOCKERY_DIR/cmd/mockery/mockery.go"
+MOCKERY_BINARY="$MOCKERY_DIR/cmd/mockery/mockery"
+
+go build -o "$MOCKERY_BINARY" "$MOCKERY_MAIN_FILE"
```

Comment:
You can invoke go binaries directly using `go run $MOCKERY_MAIN_FILE arg1 arg2 ... argN`